src/inventory/disks.go

```diff
@@ -55,8 +55,12 @@ func (d *disks) getDisks() []*models.Disk {
 		return ret
 	}
 	for _, disk := range blockInfo.Disks {
-		if disk.IsRemovable || disk.BusPath == ghw.UNKNOWN {
+		if disk.IsRemovable {
 			continue
+		} else {
+			if disk.SizeBytes == 0 || disk.DriveType == ghw.DRIVE_TYPE_UNKNOWN {
```

Comment:
Actually as I see it drive type might be misleading, because on my laptop I have partitions that are all SSDs, but we want only the disk.   But as I saw, only one is with bus-type that is not unknown.  So, it will be better to check that the bus type is not unknown.  It is also possible to see in the output of https://issues.redhat.com/browse/MGMT-1661 that it might solve the problem

Reply:
Changed.